examples/amd64/main.tf

```diff
@@ -43,4 +43,7 @@ module "this" {
   security_groups = [data.aws_security_group.this.id]
   vpc_subnets     = data.aws_subnets.this.ids
   worker_pool_id  = random_pet.this.id
+  spacelift_api_key_id = var.spacelift_api_key_id
```

Comment:
The arguments should be sorted alphabetically, and the file formatted using the `terraform fmt` command for consistency.

Reply:
Done